src/SFA.DAS.ApprenticeCommitments.Database/Tables/Revision.sql

```diff
@@ -26,6 +26,9 @@ CREATE TABLE [dbo].Revision
     [StoppedReceivedOn] DATETIME2 NULL, 
     [DeliveryModel] INT NOT NULL DEFAULT 0, 
     [EmploymentEndDate] DATETIME2 NULL, 
+    [RecognisePriorLearning] BIT NULL,
+    [DurationReducedByHours] SMALLINT NULL,
+    [DurationReducedBy] SMALLINT NULL,
```

Comment:
@Pauljgraham not sure about these SMALLINTS. They are mapped as INTS into C# and it's caused my local to expect them as INT16s. Probs safer just to make them INTS.